chapters/refereecommands.adoc

```diff
@@ -148,11 +148,17 @@ Penalty Kicks are used to punish teams that received multiple <<Yellow Card, yel
 
 ==== Yellow Card
 .Definition
-A yellow card can only be given during <<Halt, halt>>.
 
 If the yellow card is shown as a result of <<Unsporting Behavior, unsporting behavior>>, the referee may decide to immediately <<Halt, halt>> the match. In this case, the match continues with a direct free kick for the other team.
 
-Upon receipt of a yellow card, the number of robots allowed on the field for the penalized team decreases by one. If, after this decrease, the team has more robots than permitted on the field, a robot must be <<Robot Substitution, taken out>> before <<Resuming The Game, play resumes>>. The penalized team can choose the robot to remove.
+Upon receipt of a yellow card, the number of robots allowed on the field for the penalized team decreases by one. If, after this decrease, the team has more robots than permitted on the field, a robot must be <<Robot Substitution, taken out>>.
+
+After the yellow card is issued the game will not be stopped, the team will have 10 seconds to <<Robot Substitution, automatically remove the robot>>. If a robot is not taken out within time, the game is stopped for <<Robot Substitution, manual substitution>>.
```

Comment:
Let's add a note here, that the game will still be stopped for fouls, if those fouls cause a stop.

Especially the rule for dropping parts will still cause a stop, so teams are able to replace the robot.